### ISSUE TITLE
Accessibility issues - ARIA properties should be applied on hover

### ISSUE THREAD

**Issue by borzaka** [istvan-ujjmeszaros/bootstrap-dropdown-hover#21]
`aria-expanded` attribute should be toggled for accessibility.

Example from [Dropdowns docs](http://getbootstrap.com/docs/3.3/components/#dropdowns):
```
<div class="dropdown">
	<button class="btn btn-default dropdown-toggle" type="button" id="dropdownMenu1" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Dropdown <span class="caret"></span></button>
	<ul class="dropdown-menu" aria-labelledby="dropdownMenu1">
		<li><a href="#">Action</a></li>
		<li><a href="#">Another action</a></li>
		<li><a href="#">Something else here</a></li>
		<li role="separator" class="divider"></li>
		<li><a href="#">Separated link</a></li>
	</ul>
</div>
```

`aria-expanded="false"` when closed, `aria-expanded="true"` when opened.

**Comment by istvan-ujjmeszaros** [comment]
The plugin is using the Bootstrap javascript API to toggle the dropdowns, so it seems to be a Bootstrap issue, but I will look into it, thanks for the details.

**Comment by istvan-ujjmeszaros** [comment]
Just checked and it seems to work fine. Please reopen with a demo that shows the issue if you still experience it.
